dev-infrastructure/Makefile

```diff
@@ -188,7 +188,7 @@ svc.rg:
 
 svc: svc.rg
 	@scripts/cleanup-orphaned-rolebindings.sh $(SVC_RESOURCEGROUP)
-	@./ensure-no-running-deployment.sh $(SVC_RESOURCEGROUP) $(SVC_RG_DEPLOYMENT_NAME)-infra
+	@./ensure-no-running-deployment.sh $(SVC_RESOURCEGROUP) $(SVC_RG_DEPLOYMENT_NAME)-infra $(SVC_KV_RESOURCEGROUP)
```

Comment:
not sure about this third argument to the script. can you elaborate?

Reply:
Ooops, yeah that's not right.  Should be fixed.